content/blog/2020-09-17-support-for-new-jsx-transform.md

```diff
@@ -0,0 +1,244 @@
+---
+title: "React v17.0 Release Candidate: Support for New JSX Transform"
```

Comment:
Maybe "Introducing the New JSX Transform"? I don't think this is very related to 17 RC and might be confusing to see two posts that say they're about 17 RC.